infra/config-store/src/main/java/com/simprints/infra/config/store/local/migrations/ProjectConfigFaceSdkQualityThresholdMigration.kt

```diff
@@ -0,0 +1,36 @@
+package com.simprints.infra.config.store.local.migrations
+
+import androidx.datastore.core.DataMigration
+import com.simprints.infra.config.store.local.models.ProtoProjectConfiguration
+import com.simprints.infra.logging.Simber
+import javax.inject.Inject
+
+/**
+ * Can be removed once all the devices have been updated to 2022.4.1
+ */
+internal class ProjectConfigFaceSdkQualityThresholdMigration @Inject constructor(
+) : DataMigration<ProtoProjectConfiguration> {
+
+    override suspend fun cleanUp() {
+        Simber.i("Migration of device configuration to Datastore done")
+    }
+
+    override suspend fun shouldMigrate(currentData: ProtoProjectConfiguration): Boolean =
+        currentData.face?.rankOne?.qualityThreshold?.let { it > 0 } == true
```

Comment:
Hm, what if the currently value is 0? Like in GG2 prod :D

Reply:
It would still be 0 in the new field as it is the number's default value.